app/build.gradle

```diff
@@ -69,7 +69,7 @@ dependencies {
     implementation "androidx.lifecycle:lifecycle-runtime-ktx:2.5.1"
     implementation 'androidx.window:window:1.1.0-alpha03'
 
-    def camerax_version = '1.2.0-rc01'
+    def camerax_version = '1.2.0-beta01'
```

Comment:
Hi @tibbi 
I reverted this back because VideoCapture fails to initialize on some of my devices when we on updating to [`1.2.0-rc01`](https://developer.android.com/jetpack/androidx/releases/camera#1.2.0-rc01), video capture fails to configure on Samsung A52s (Android 12) but works on Samsung S21 (Android 12) and Pixel 7 Pro (Android 13). Even the [`1.3.0-alpha01`](https://developer.android.com/jetpack/androidx/releases/camera#1.3.0-alpha01)doesn’t work. The last working version was 1.2.0-beta01. I would downgrade and check the [change logs](https://android.googlesource.com/platform/frameworks/support/+log/52096c49378ea748c1b11b54c04c19c8d1e571c6..4492a75904152396b00e333181677981b0e28fd0/camera?s=759f968ff550847ed2bde3f23e979d465a5dd8f6) to see what has changed. 

I noticed that these versions work on using the [CameraX Video sample app](https://github.com/android/camera-samples/tree/main/CameraXVideo)